### PR TITLE
chore(artifacts|cli): move `cli` into `artifacts`

### LINKED ISSUE #88: cover CLI usage in README

### LINKED ISSUE #98: Merge `cli` and `artifacts` packages
**Description**

The two packages `@zk-kit/artifacts-cli` and `@zk-kit/artifacts` serve very similar purposes and could be merged in one package. In particular, `@zk-kit/artifacts` could integrate the code of the CLI package.

### PR DESCRIPTION
**Pull request by sripwoud**
Closes #98
Closes #88

### CODE REVIEW

File: packages/artifacts/rollup.config.ts [hunk 1]
```diff
@@ -16,6 +16,17 @@ const banner = `/**
...
+
+/**
+ * @module ${pkg.name}
+ * @version ${pkg.version}
+ * @file ${pkg.description}
+ * @copyright Ethereum Foundation ${new Date().getFullYear()}
+ * @license ${pkg.license}
+ * @see [Github]{@link ${pkg.homepage}}
+*/
+`
```
Comment (cedoor) [inline packages/artifacts/rollup.config.ts:29]: This string is the same as the one above, I guess we can re-use it.

File: packages/artifacts/tsconfig.build.json [hunk 2]
```diff
@@ -1,8 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "compilerOptions": {
-    "declaration": true,
-    "outDir": "dist/types"
-  },
```
Comment (cedoor) [inline packages/artifacts/tsconfig.build.json:6]: Why have these options been removed?

File: packages/artifacts/tsconfig.json [hunk 3]
```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src", "test", "rollup.config.ts"]
+  "include": ["src", "test", "rollup.config.ts"],
+  "compilerOptions": {
+    "baseUrl": "src"
```
Comment (cedoor) [inline packages/artifacts/tsconfig.json:5]: It's working but I think this option should be copied in the `tsconfig.build.json` file too.

**Review by cedoor — COMMENTED** [review]
In the `exports` field (`packages/artifacts/package.json` file), `types` should have the browser types, and `default` should have the browser file.

File: packages/artifacts/tsconfig.build.json (hunk 2 above)
Comment (sripwoud) [reply packages/artifacts/tsconfig.build.json]: hmm, honestly I don't remember

File: packages/artifacts/rollup.config.ts (hunk 1 above)
Comment (sripwoud) [reply packages/artifacts/rollup.config.ts]: https://github.com/privacy-scaling-explorations/snark-artifacts/pull/118/commits/31c7bab085bcc3eab2d575dc3f287a7047329a92

**Review by cedoor — APPROVED** [review]

**Pull request merged**